package.json

```diff
@@ -43,6 +43,7 @@
     "@react-native-community/async-storage": "^1.8.1",
     "@react-native-community/geolocation": "^2.0.2",
     "@react-native-community/masked-view": "0.1.5",
+    "@react-native-community/netinfo": "^5.7.0",
```

Comment:
what's the reason for adding netinfo?

Reply:
It is for network check. I have added another issue which was 256 which was not merged. I will remove this also.

Reply:
Removed